### PR TITLE
MAINT: Fix some typos in a code string and comments

### PR DESCRIPTION
**Pull request by dongjoon-hyun**

### CODE REVIEW

**Comment by njsmith** [comment]
Wow, thank you!

@homu: r+

**Comment by njsmith** [comment]
Huh, those are some weird failures that travis is throwing up :-/ I don't see how they can possibly be related to this PR, but does anyone know what's going on? that's 4 errors over 2 builds, 3 different error messages :-/

**Comment by dongjoon-hyun** [comment]
In `homu` tests, I found the following error. Is that a reason?
```
sudo: debootstrap: command not found
```

**Comment by dongjoon-hyun** [comment]
In `travis-ci-/pr`, I found another error.
```
./tools/travis-test.sh: line 61: python3-dbg: command not found
```

**Comment by dongjoon-hyun** [comment]
I think I found the reason. Travis CI fails to install the required software due to network traffics (maybe the aftereffect of today's GitHub errors).
```
3.5 is not installed; attempting download
bzip2: Compressed file ends unexpectedly;
	perhaps it is corrupted?  *Possible* reason follows.
bzip2: Inappropriate ioctl for device
	Input file = (stdin), output file = (stdout)
It is possible that the compressed file(s) have become corrupted.
You can use the -tvv option to test integrity of such files.
You can use the `bzip2recover' program to attempt to recover
data from undamaged sections of corrupted files.
tar: Unexpected EOF in archive
tar: Unexpected EOF in archive
tar: Error is not recoverable: exiting now
```

File: numpy/f2py/rules.py [hunk 1]
```diff
@@ -15,14 +15,14 @@
     get_b_from_python
     if (successful) {
 
-      callfortran
-      if (succesful) {
+      call_fortran
```
Comment (rgommers) [inline numpy/f2py/rules.py]: This one shouldn't be changed I think - ``callfortran`` is used a lot in the rest of this file.

**Comment by rgommers** [comment]
nice cleanup, thanks @dongjoon-hyun

**Comment by charris** [comment]
Github went down hard for about five hours at ~5:20 PM MST yesterday  and it took a while to recover.

**Comment by dongjoon-hyun** [comment]
Oh, thank you for merging!

**Comment by rgommers** [comment]
@njsmith I don't like that auto-merge trick - having a bot as commit author is just wrong. plus practical reasons, like it's confusing and messes with commits counts etc.

**Comment by njsmith** [comment]
Hmm, not sure what happened here this time -- I feel like it should have given up once the first build failed.

(Did someone hit the "rerun this build" button on Travis or something?)

The useful bits about the bot are that (1) it gives a way to say "merge if tests pass" sitting around and waiting for them, which is why I used it this time, and (2) by testing before merging it guarantees that master always passes tests (because -- unlike normal Travis runs -- it actually tests the merge that's being committed, instead of some merge with whatever matter looked at when the PR was first submitted, maybe weeks ago). There are a number of projects that use it exclusively for these reasons (esp. in the rust ecosystem).

The annoying things are, it's too chatty, and it doesn't support appveyor, so meh. I guess I could add it not setting the "committed by" field properly to my mental list of grievances? I'm not sure what to do with it being "just wrong" and "confusing" -- it definitely solves some real technical problems for some cases -- and I feel like I care more about prompt and rigorous testing than about commit counting tools, but yeah, it's definitely not an obvious 100% win :-/. Maybe with some fixes it could be...

**Comment by rgommers** [comment]
> I'm not sure what to do with it being "just wrong" and "confusing" -- it definitely solves some real technical problems for some cases -- 

Well, in this case it merged an open PR *after* I had made a comment on it. That confused me (I only noticed because of the "thanks for merging" comment), and is definitely not the right thing to do.

And ``@homu`` is clearly confusing the people that keep on replying to it:)

**Comment by njsmith** [comment]
Yeah, like I said I have no idea why it did that :-/ the logic is supposed
to be, if someone approves the PR then it runs the tests, and either merges
or not depending on the outcome. Somehow here it ran the tests, they
failed, it announced that it wasn't merging, and then it ran them again (??
or something caused them to be rerun), and this time they passed so it
merged?
.
@barosl: any idea what might be going on here?

**Comment by charris** [comment]
On Fri, Jan 29, 2016 at 2:49 AM, Nathaniel J. Smith <
notifications@github.com> wrote:

> Yeah, like I said I have no idea why it did that :-/ the logic is supposed
> to be, if someone approves the PR then it runs the tests, and either merges
> or not depending on the outcome. Somehow here it ran the tests, they
> failed, it announced that it wasn't merging, and then it ran them again (??
> or something caused them to be rerun), and this time they passed so it
> merged?
>

I reran the tests. When github came back on line tests failed because not
all of the network was yet up, so I reran them.

Chuck

**Comment by dongjoon-hyun** [comment]
Hi, all, Sorry for interrupting your discussion.
According to the context, I think it's accidentally merged during under review and the following should be fixed. Am I right? :)
```
This one shouldn't be changed I think - callfortran is used a lot in the rest of this file.
Add a line note
```
How can I recover the `callfortran` stuff? Should I make another PR for this that line? Please give me some advice.

**Comment by rgommers** [comment]
@dongjoon-hyun no worries. A new PR for that would be good.

**Comment by dongjoon-hyun** [comment]
Okay, thank you, @rgommers . I'll make a new `MAINT` PR for that.
```
-      call_fortran
+      callfortran
```

**Comment by njsmith** [comment]
@charris: Ahhh, okay, at least now we know what happened :-) Pretty weird edge case here...

**Pull request merged**